components/Modal.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="modal-wrapper">
+    <disable-modal
+      :is-open="open === 'disable'"
+      :resource="data"
+      @close="close"
+    />
+    <report-modal
+      :is-open="open === 'report'"
+      :resource="data"
+      @close="close"
+    />
+  </div>
+</template>
+
+<script>
+import DisableModal from '~/components/Modal/DisableModal'
+import ReportModal from '~/components/Modal/ReportModal'
```

Comment:
I don’t think that importing all modals in one component is a good idea.

Reply:
But as they are used mostly everywhere it’s okay for now, but when we start adding more specific modals, this will be not very efficient.

Reply:
@roschaefer